### DISCUSSION TITLE
Break-even fee ratio for multisig redeem

### DISCUSSION THREAD

**Discussion by Jpja** [CounterpartyXCP/CIPs#93] [General]
This transaction redeemed 5 multisig outputs of 7800 sats each:
[https://blockstream.info/tx/58ecdf02ac1d299b5d59d81f92c22ab1195377a60f13ae8d080a0d7a7ddb8157](https://blockstream.info/tx/58ecdf02ac1d299b5d59d81f92c22ab1195377a60f13ae8d080a0d7a7ddb8157)

From a total dust amount of 39000 sats, 15000 sats were paid in fees and 24000 were redeedmed.
The fee ratio was 24.6 sat/B, and with some quick math it would have been economical to redeem up until **63.4 sat/B**.

With talks of reducing the dust amount the 1000 sats, the break even point will be 7.8x less: **8.1 sat/B**

_This example used 5 inputs. Would be good to do a similar calculation where only a single multisig was swept._

**Comment by jotapea** [comment]
Talks? I see it was just [merged](https://github.com/CounterpartyXCP/counterparty-lib/commit/71ff5f0b953237cd72012c111ec102954aa170f0) to master.

First, we need multiple reviewers before merging to master.

Now on the topic, It might seem like nobody would complaint about this change, but I've always thought that having minimal sats per multisig output is counterproductive. And it is because of the point @Jpja is bringing up, people should be incentivized to reclaim those sats!

Until more discussions are done on the topic, I would say to revert this merge to master. I actually believe leaving the higher amount as a protocol default is PRO Bitcoin, because of the alignment of incentives to clean up these UTXOs.

And anyone can still modify the values manually if they wish.

**Comment by jotapea** [comment]
https://github.com/CounterpartyXCP/counterparty-lib/commit/71ff5f0b953237cd72012c111ec102954aa170f0#commitcomment-119462529
